modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -3,27 +3,33 @@
 // * osd_planning/aws-ccs.adoc
 // * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc
 // * rosa_planning/rosa-sts-aws-prereqs.adoc
-// * rosa_planning/rosa-hcp-prereqs.adoc
 
 ifeval::["{context}" == "rosa-sts-aws-prereqs"]
 :fedramp:
 endif::[]
-ifeval::["{context}" == "rosa-hcp-aws-prereqs"]
-:fedramp:
-endif::[]
 
 :_mod-docs-content-type: PROCEDURE
+ifeval::["{context}" == "rosa-sts-aws-prereqs"]
+[id="rosa-classic-firewall-prerequisites_{context}"]
```

Comment:
I could be wrong don't think this is working correctly. 

![image](https://github.com/openshift/openshift-docs/assets/16167833/613eb1ad-7004-4625-9661-5f8b786df2ca)

I believe that it should be `ifdef::openshift-rosa[]`. If we want to create the conditional for this assembly, we need to add an ifeval at the top and bottom of the module, with some term in the middle like `:rosa-classic:`

```
ifeval::["{context}" == "rosa-sts-aws-prereqs"]
:rosa-classic:
endif::[]
```

Check out this module `/modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc` for some examples